### PR TITLE
Makes sock fork clean & without conflict

### PR DESCRIPTION
**Pull request by caesar2164**
PR to make fork clean, we should also discuss a bigger upstream PR.

### CODE REVIEW

File: cms/templates/widgets/sock.html [hunk 1]
```diff
@@ -47,12 +47,14 @@ <h2 id="sock-heading" class="title sr-only">${_("{studio_name} Documentation").f
             'sr_mouseover_text': _('Send an email to {email}').format(email=partner_email),
             'text': _('Contact Us'),
             'condition': bool(partner_email)
-        }, {
+        }]
+
+        links.append({
```
Comment (stvstnfrd) [inline cms/templates/widgets/sock.html]: This isn't a change we can push upstream, as is.
How can we move this into a template/fragment?

Comment (caesar2164) [reply cms/templates/widgets/sock.html]: thus why we need to discuss...

Comment (caesar2164) [reply cms/templates/widgets/sock.html]: because the change could move these dicts into settings, python, separate mako file, etc...

**Comment by stvstnfrd** [comment]
retest this please

File: cms/templates/widgets/sock.html [hunk 2]
```diff
@@ -18,41 +18,7 @@
...
-            'sr_mouseover_text': _('Send an email to {email}').format(email=partner_email),
-            'text': _('Contact Us'),
-            'condition': bool(partner_email)
-        }, {
-            'href': 'https://stanfordonline.zendesk.com/hc',
-            'sr_mouseover_text': _('Find answers in the FAQ or ask a new question.'),
-            'text': _('Help Center'),
-            'condition': True
-        }]
+          links = settings.STUDIO_SOCK_LINKS
```
Comment (stvstnfrd) [inline cms/templates/widgets/sock.html]: We don't need this variable, nor the entire code block.
Instead, just loop over the settings variable directly.

```
% for link in settings.STUDIO_SOCK_LINKS:
    <!-- whatever -->
% endfor
```

File: cms/envs/common.py [hunk 3]
```diff
@@ -1238,6 +1239,35 @@
...
+        'sr_mouseover_text': _('Enroll in edX101: Overview of Creating an edX Course'),
+        'text': _('Enroll in edX101'),
+        'condition': True
+    }, {
+        'href': 'https://www.edx.org/course/creating-course-edx-studio-edx-studiox',
+        'sr_mouseover_text': _('Enroll in StudioX: Creating a Course with edX Studio'),
+        'text': _('Enroll in StudioX'),
+        'condition': True
+    #    }, {
+    #        'href': 'mailto:{email}'.format(email=PARTNER_SUPPORT_EMAIL),
```
Comment (stvstnfrd) [inline cms/envs/common.py]: Maybe try uncommenting this block,
starting by removing the variables/formatting.
Make sure that passes, then add the variables line by line.
With each change, run the test suite locally.

**Comment by stvstnfrd** [comment]
@caesar2164 should this be reworked based on the upstream PR?

**Comment by caesar2164** [comment]
Yes, but actually we should wait until it's merged upstream and just cherry-pick it...

**Pull request closed without merging**